src/test/kotlin/three/consulting/epoc/service/TimesheetEntryServiceIntegrationTest.kt

```diff
@@ -322,4 +322,29 @@ class TimesheetEntryServiceIntegrationTest : IntegrationTest() {
         assertThat(csv.contains("7.5;test;test;2022-04-01;$email")).isTrue
         assertThat(csv.contains("7.5;test;test;2022-04-03;test@worker.fi")).isFalse
     }
+
+    @Test
+    fun `timesheet entries for csv export order exists`() {
+        val date = LocalDate.parse("2023-03-01")
+        val desc = "Testing timesheet entry5"
+        val entry = timesheetEntryService.findTimesheetEntryForId(5)
+        assertThat(entry).isNotNull
+        if (entry != null) {
+            assertThat(entry.description).isEqualTo(desc)
+            assertThat(entry.date).isEqualTo(date)
+        }
```

Comment:
Always avoid optional blocks in test assertions, even if you have the nonnull verified before. This can just be 
```kotlin
assertThat(entry).isNotNull
assertThat(entry?.description).isEqualTo(desc)
assertThat(entry?.date).isEqualTo(date)
```

Reply:
Though with this the first `.isNotNull` is kinda redundant, as the following two will reveal if things are null

Reply:
Good point. I didn't think when I did this.